docs/source/markdown/podman-machine-init.1.md.in

```diff
@@ -96,6 +96,13 @@ if there is no existing remote connection configurations.
 
 API forwarding, if available, follows this setting.
 
+### **--run-playbook**
+
+Add the provided Ansible playbook to the machine and execute it after the first boot.
+
+Note: The playbook will be executed with the same privileges given to the user in the virtual machine
```

Comment:
Maybe make clear that this playbook cannot include other files from the local system as we only copy the one file.